README.md

```diff
@@ -49,35 +49,37 @@ And you are good to go! You can start [exploring our tutorial notebooks](https:/
 **Inductiva API** contains pre-built scenarios that define physical systems of interest ready to simulate. Users can choose some parameters and configure the system according to their needs, run the simulation using the most adequate resources and visualize the results.
 
 
-### Example
+### WindTunnel Example
 
 ```python
- import inductiva
+import inductiva
 
 inductiva.api_key = "YOUR_API_KEY"
 
-# Download the insulin protein (ID - "1ZNI") from RCSB database
-insulin_pdb_file = inductiva.molecules.utils.download_pdb_from_rcsb(pdb_id="1ZNI")
-
- # Initialize the scenario
- scenario = inductiva.molecules.ProteinSolvation(
-     protein_pdb = insulin_pdb_file,
-     temperature = 300)
+# Url to a test object in Inductiva Github repository
+object_url = "https://raw.githubusercontent.com/inductiva/inductiva/main" \
```

Comment:
(minor): I mentioned this in a previous review: why don't you call it vehicle.obj ?